.ci/openshift-ci/images/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build the image which wraps the kata-containers installation along with the
+# install script. It is used on a daemonset to deploy kata on OpenShift.
+#
+FROM centos:8
+
+RUN yum install -y rsync
```

Comment:
Nit: `dnf` :smile:

Reply:
Not on `centos` image.

```
[root@f5018106fe91 /]# yum
Loaded plugins: fastestmirror, ovl
You need to give some command
Usage: yum [options] COMMAND

[root@f5018106fe91 /]# dnf
bash: dnf: command not found
```

Reply:
> Looks good to me. Maybe wait for @jodh-intel to ack the replies we gave. I think that @devimc is not available this week.

Thanks @c3d ! I am about to send another version with two small changes as suggested by @jodh-intel . Then we will be ready to go :)

Reply:
Ah - you get it by default, but clearly not with your image ;)

Reply:
> Not on `centos` image.
> 
> ```
> [root@f5018106fe91 /]# yum
> Loaded plugins: fastestmirror, ovl
> You need to give some command
> Usage: yum [options] COMMAND
> 
> [root@f5018106fe91 /]# dnf
> bash: dnf: command not found
> ```

hmm... interesting.

On v4 of this PR I changed to centos:7 and the reason is: this image will be built in a OpenShift Build process which usually replaces the `FROM` to whatever the user choose. In the case of this CI, it will use RHEL-7 so the closes we got (for the sake of debugging/development in your workstation) is centos-7.

Reply:
Ah - right. I'd missed the downgrade to centos 7 ;)